h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -49,6 +50,9 @@
  * Generalized linear model implementation.
  */
 public class GLM extends ModelBuilder<GLMModel,GLMParameters,GLMOutput> {
+
+  private static final Logger LOG = Logger.getLogger(GLM.class);
```

Comment:
This class is already using `water.util.Log` to handle logging

We should stick with one - I think in this PR you probably don't want to redo everything. cc: @honzasterba

Reply:
Log4j removed and water.util.Log is used